### DISCUSSION TITLE
✨ immich

### DISCUSSION THREAD

**Discussion by glitchert** [runtipi/runtipi-appstore#3652] [Requests]
### Description

Self-hosted photo and video management solution, with phone backup.

### Official Website

https://immich.app/

### Documentation link

https://immich.app/docs/overview/introduction/

### Last application release & date

1.105.1 (15/5/2024)

### Application license

AGPL-3.0 license

### Source code repository link

https://github.com/immich-app/immich

### Docker image link

ghcr.io/immich-app/immich-server:main

### Other's

I noticed there was no photo management with Ai apps and wanted to suggest this one,
especially because this app and Runtipi has the same UI/UX sensibilities/feel.

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote ⬆️ this discussion to show your interest in this request!

This is the newest management and phone backup photos app, with Ai and natural search.
+modern UI.

![immich-screenshots](https://github.com/runtipi/runtipi-appstore/assets/59421327/1a423022-c8b6-4da7-bf83-466dfb7709ee)

Thanks ⛺

**Comment by glitchert** [comment]
Sorry, I thought what is available in the demo is all the apps there is,
But after browsing the app-available section, I found immich there.

**Reply by jasonpearce** [reply]
Yes. Immich is already an available app. I recommend this request changed to closed.

**Reply by JigSawFr** [reply]
@jasonpearce Thanks for help !
